includes/install.php

```diff
@@ -219,19 +220,20 @@ function give_on_create_blog($blog_id, $user_id, $domain, $path, $site_id, $meta
  */
 function give_wpmu_drop_tables($tables, $blog_id)
 {
+    global $wpdb;
+
     switch_to_blog($blog_id);
-    $custom_tables = __give_get_tables();
 
-    /* @var Give_DB $table */
-    foreach ($custom_tables as $table) {
-        if ($table->installed()) {
-            $tables[] = $table->table_name;
-        }
+    $prefix = $wpdb->prefix . 'give_%';
+    $giveTables = $wpdb->get_col("SHOW TABLES LIKE '{$prefix}'");
```

Comment:
Is this a fix to a problem or just a performance enhancement? From what I can tell, this reduces the number of queries, but otherwise I don't see any difference in the result. Just clarifying.

Reply:
@kjohnson The previous version did not list all tables, resulting in some of them being left behind.